### ISSUE TITLE
Redirect issue

### ISSUE THREAD

**Issue by erikbelusic** [M1ck0/adskipper-extension#27]
I think youtube has detected and worked around this again. On about 50% of the videos, I now see the "you are using ad adblocker" message and i end up in an infinite redirect loop back to the video to see the message again, over and over

```
ReferenceError: removePageAds is not defined at stopYoutubeAd (chrome-extension://dinhbmppbaekibhlomcimjbhdhacoael/scripts/youtube.js:205:5) at chrome-extension://dinhbmppbaekibhlomcimjbhdhacoael/scripts/youtube.js:331:13 at Array.forEach () at MutationObserver.observerCallback (chrome-extension://dinhbmppbaekibhlomcimjbhdhacoael/scripts/youtube.js:327:15)
```

**Comment by AdityaDas5501** [comment]
Absolutely

**Comment by M1ck0** [comment]
Logic is changed in the version 4.1 (pending on Chrome and it is live on Firefox) so please check it out as soon as possible, and open new issue if this happens again.

Thank you so much for understanding, and sorry for late reply

**Comment by AdityaDas5501** [comment]
> Logic is changed in the version 4.1 (pending on Chrome and it is live on Firefox) so please check it out as soon as possible, and open new issue if this happens again.
> 
> Thank you so much for understanding, and sorry for late reply

When will it be live on Chrome, I'm waiting eagerly

**Comment by M1ck0** [comment]
@AdityaDas5501 It is live. Go check if you have version 4.1 installed

**Comment by AdityaDas5501** [comment]
> @AdityaDas5501 It is live. Go check if you have version 4.1 installed

What?! It's still showing me version 3.9 in Chrome webstore and the extension still isn't updated
![image](https://github.com/user-attachments/assets/16eca7a6-a1d1-4407-acdf-59e2375f3475)

Help 😢
